File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/inplaceadd_runme.m
inplaceadd
a = inplaceadd.A(7);

a += 5;
if (a.val != 12)
  error
endif

a -= 5;
if a.val != 7:
  error
endif

a *= 2;

if (a.val != 14)
  error
endif

a += a;
if (a.val != 28)
  error
endif

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/inplaceadd_runme.py
import inplaceadd
a = inplaceadd.A(7)

a += 5
if a.val != 12:
    print a.val
    raise RuntimeError

a -= 5
if a.val != 7:
    raise RuntimeError

a *= 2

if a.val != 14:
    raise RuntimeError

a += a
if a.val != 28:
    raise RuntimeError
